src/cachingreaderworker.cpp

```diff
@@ -7,24 +7,23 @@
 #include "cachingreaderworker.h"
 #include "trackinfoobject.h"
 #include "soundsourceproxy.h"
-#include "sampleutil.h"
 #include "util/compatibility.h"
 #include "util/event.h"
 #include "util/math.h"
 
-// There's a little math to this, but not much: 48khz stereo audio is 384kb/sec
-// if using float samples. We want the chunk size to be a power of 2 so it's
-// easier to memory align, and roughly 1/2 - 1/4th of a second of audio. 2**17
-// and 2**16 are nice candidates. 2**16 is 170ms of audio, which is well above
-// (hopefully) the latencies people are seeing. at 10ms latency, one chunk is
-// enough for 17 callbacks. We may need to tweak this later.
-
-// Must be divisible by 8, 4, and 2. Just pick a power of 2.
-#define CHUNK_LENGTH 65536
-
-const int CachingReaderWorker::kChunkLength = CHUNK_LENGTH;
-const int CachingReaderWorker::kSamplesPerChunk = CHUNK_LENGTH / sizeof(CSAMPLE);
 
+// One chunk should contain 1/2 - 1/4th of a second of audio.
+// 8192 frames contain about 170 ms of audio at 48 kHz, which
+// is well above (hopefully) the latencies people are seeing.
+// At 10 ms latency one chunk is enough for 17 callbacks.
+// Additionally the chunk size should be a power of 2 for
+// easier memory alignment.
+// TODO(XXX): The optimum value of the "constant" kFramesPerChunk
```

Comment:
Don't know why the master commits appear on this branch!? I will reset my branch to my last commit and again merge from master.